sources/engine/Stride.Graphics/SDL/Window.cs

```diff
@@ -342,6 +342,28 @@ public Size2 Size
             }
             set { SDL.SetWindowSize(sdlHandle, value.Width, value.Height); }
         }
+        
+        /// <summary>
```

Comment:
XML grammar improvements

```
/// <summary>
/// The opacity of the window.
/// </summary>
/// <remarks>The value should be between 0.0f and 1.0f. It will automatically be clamped to this range.</remarks>
```